packages/core/src/components/panel-stack/panelStack.tsx

```diff
@@ -46,6 +46,12 @@ export interface IPanelStackProps extends IProps {
      */
     onOpen?: (addedPanel: IPanel) => void;
 
+    /**
+     * If false, PanelStack will render all panels in the stack to the DOM, allowing their React component trees to maintain state as a user navigates through the stack. Panels other than the currently active one will be invisible.
+     * @default true
+     */
+    renderCurrentPanelOnly?: boolean;
```

Comment:
let's change this name to mirror a similar prop for the `Tabs` component... `renderActivePanelOnly`